src/transport/SecurePairingSession.h

```diff
@@ -74,6 +74,17 @@ class DLL_EXPORT SecurePairingSessionDelegate
 
 struct SecurePairingSessionSerialized;
 
+typedef struct SecurePairingSessionSerializable
```

Comment:
Why not just `struct SecurePairingSessionSerializable` instead of this C-style typedef?